frontend/src/components/RecentOperations/index.tsx

```diff
@@ -12,7 +12,11 @@ import { OperationTypeColumn } from "./OperationTypeColumn";
 import { fetchLastOperationsAction } from "ducks/operations";
 import { FetchLastOperationsActionResponse, Network } from "types";
 
-export const RecentOperations = () => {
+export const RecentOperations = ({
+  network = Network.MAINNET,
+}: {
+  network?: Network;
+}) => {
   const { operations } = useRedux("operations");
   const dispatch = useDispatch();
   const horizonURL = "https://horizon.stellar.org";
```

Comment:
Not part of this PR, why do we hard code Horizon URL here instead of getting it from `networkConfig` in `constants/settings.ts`?

Reply:
sorry, I took as base the dashboards v1, I will fix it